### PR TITLE
fix(bindableswithcompat, shapes): [android,wasm] Color brushes added …

### PR DESCRIPTION
**Pull request by NicolasChampagne**
SolidColorBrushes added to Thumbtint, TrackTint (BindableSwitchCompat for Android), Fill, and Stroke (Shapes for Wasm) missing opacity

GitHub Issue (If applicable): #3125 #3126

## PR Type

What kind of change does this PR introduce?

## What is the current behavior?

SolidColorBrushed that contained Opacity when added 

## PR Checklist

Please check if your PR fulfills the following requirements:

~~- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)~~
~~- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)~~
~~- [ ] Validated PR `Screenshots Compare Test Run` results.~~
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/EllipsePage.xaml [hunk 1]
```diff
@@ -7,6 +7,16 @@
 	mc:Ignorable="d"
 	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
+	<Page.Resources>
```
Comment (jeromelaban) [inline src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/EllipsePage.xaml:10]: Why did you change that file ? Is it for another PR ?

Comment (NicolasChampagne) [reply src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/EllipsePage.xaml:10]: I added two Ellipses in this file, one with a Fill that uses a SolidColorBrush with opacity and the other a Stroke that uses one. To verify that shapes in WebAssembly function with the changes as well.
